modules/discord/meta.nix

```
@@ -1,5 +1,11 @@
 { lib, ... }:
 {
-  maintainers = [ lib.maintainers.Flameopathic ];
   name = "Discord";
+  homepages = {
+    Discord = "https://discordapp.com/";
+    Vencord = "https://github.com/Vendicated/Vencord";
+    Vesktop = "https://github.com/Vencord/Vesktop";
+    Nixcord = "https://github.com/KaylorBen/nixcord";
```

Consider sorting `homepages` entries:

```suggestion
    Discord = "https://discordapp.com/";
    Nixcord = "https://github.com/KaylorBen/nixcord";
    Vencord = "https://github.com/Vendicated/Vencord";
    Vesktop = "https://github.com/Vencord/Vesktop";
```